ipywidgets/widgets/widget_float.py

```diff
@@ -141,6 +174,48 @@ class FloatSlider(_BoundedFloat):
     continuous_update = Bool(True, help="Update the value of the widget as the user is holding the slider.").tag(sync=True)
     disabled = Bool(False, help="Enable or disable user changes").tag(sync=True)
 
+    style = InstanceDict(SliderStyle).tag(sync=True, **widget_serialization)
+
+
+@register
+class FloatLogSlider(_BoundedLogFloat):
+    """ Slider/trackbar of logarithmic floating values with the specified range.
+
+    Parameters
+    ----------
+    value : float
+        position of the slider
+    base : float
+        base of the logarithmic scale. Default is 2
+    min : float
+        minimal position of the slider in log scale, i.e., actual minimum is base ** min
+    max : float
+        maximal position of the slider in log scale, i.e., actual maximum is base ** max
+    step : float
+        step of the trackbar, denotes steps for the exponent, not the actual value
+    description : str
+        name of the slider
+    orientation : {'horizontal', 'vertical'}
+        default is 'horizontal', orientation of the slider
+    readout : {True, False}
+        default is True, display the current value of the slider next to it
+    readout_format : str
+        default is '.2f', specifier for the format function used to represent
+        slider value for human consumption, modeled after Python 3's format
+        specification mini-language (PEP 3101).
+    """
+    _view_name = Unicode('FloatLogSliderView').tag(sync=True)
+    _model_name = Unicode('FloatLogSliderModel').tag(sync=True)
+    step = CFloat(0.1, help="Minimum step in the exponent to increment the value").tag(sync=True)
+    orientation = CaselessStrEnum(values=['horizontal', 'vertical'],
+        default_value='horizontal', help="Vertical or horizontal.").tag(sync=True)
+    readout = Bool(True, help="Display the current value of the slider next to it.").tag(sync=True)
+    readout_format = NumberFormat(
+        '.2f', help="Format for the readout").tag(sync=True)
+    continuous_update = Bool(True, help="Update the value of the widget as the user is holding the slider.").tag(sync=True)
+    disabled = Bool(False, help="Enable or disable user changes").tag(sync=True)
+    base = CFloat(2., help="Base for the logarithm").tag(sync=True)
```

Comment:
I know we had this discussion before - what do you think about defaulting to base 10? I think that might be more common in everyday use.

Reply:
If we did change it to base 10 (I think we probably ought to), we probably ought to default the max to something like 4, so we only go up to 10k.